jenkins-s2i/plugins.txt

```diff
@@ -0,0 +1 @@
+http_request:1.8.21
```

Comment:
Additional plugins are required. Had to install the following:

* github-branch-source:2.3.2
* kubernetes:1.2

Reply:
@sabre1041 hmm.. i did not need to make either of these changes. what version of openshift/jenkins image are you working off of?

Reply:
Just adding my $0.02, I did not have to add any additional plugins when I tested this.

Reply:
@etsauer @pabrahamsson retested and confirmed that no additional plugins are needed